packages/benchmark-test/test/benchmark.test.js

```diff
@@ -0,0 +1,58 @@
+import '@endo/init';
+import { passStyleOf } from '@endo/pass-style';
+import { test, benchmark } from '../tools/benchmark.js';
```

Comment:
Github is supposed to be better at detecting file rename and motion as different for purposes of review than a deletion and addition. Oh well. In any case, I think this is the only line I changed from your original.